tests/resources/integration_test_data/assetized_child_ref_non_assetized_grandchild/parking_lot.usd

```diff
@@ -0,0 +1,11 @@
+#usda 1.0
+
+def "ParkingLot"
+{
+    def "ParkingLot_Floor_1" (
+        references = @floor@</ParkingLot_Floor>
```

Comment:
[fix] All the assetized refs would need to be `@bal:///floor@...` otherwise BAL will ignore them.

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/6a194ba8a1a6a29c67d4d722e3d1c8f224b45c71

Arguably i'm not prescribing BAL as neccesarily the manager for this dataset ... but that argument is bunk because that's totally my intent.